src/etc/inc/filter.lib.inc

```diff
@@ -556,7 +558,7 @@ function filter_core_rules_system($fw, $defaults)
                 'to' => '(self)',
                 'to_port' => implode(' ', $lockoutprts),
                 'descr' => 'anti-lockout rule',
-                '#ref' => 'system_advanced_firewall.php#noantilockout'
+                '#ref' => 'interfaces.php?if=' . $lockoutif
```

Comment:
I don't think that makes sense as both could be valid.

Reply:
true, it was a left over from previous work. reverted